server.js

```diff
@@ -99,12 +110,12 @@ app.get('/plotType/:type', async (req, res) => {
 });
 
 app.get('/Summary', (req, res) => {
-    generateSummary().then(res.json.bind(res));
+    generateSummary(output[req.query.dir]).then(res.json.bind(res));
 });
 
 app.get('/data/basicParameters', (req, res) => {
-    output.read_para().then(() => {
-        res.json(output.parameters);
+    output[req.query.dir].read_para().then(() => {

```

Comment:
The default behaviour of query string parser will decode percent-encoded strings. Inconsistency occurs when the path contains special characters need to be encoded, e.g. '\\'. `encodeURI` this string should resolve the problem.

Reply:
This bug was fixed, please try it on a windows